components/brave_sync/brave_sync_service_unittest.cc

```diff
@@ -530,43 +533,7 @@ TEST_F(BraveSyncServiceTest, OnDeleteDeviceWhenSelfDeleted) {
   EXPECT_FALSE(DevicesContains(devices_final.get(), "2", "device2"));
 }
 
-TEST_F(BraveSyncServiceTest, OnResetSync) {
-  EXPECT_CALL(*sync_client(), OnSyncEnabledChanged).Times(AtLeast(1));
-  EXPECT_CALL(*observer(), OnSyncStateChanged(sync_service()))
-      .Times(AtLeast(3));
-  sync_service()->OnSetupSyncNewToSync("this_device");
-  EXPECT_TRUE(
-      profile()->GetPrefs()->GetBoolean(brave_sync::prefs::kSyncEnabled));
-  brave_sync_prefs()->SetThisDeviceId("0");
-
-  RecordsList records;
-  records.push_back(
-      SimpleDeviceRecord(SyncRecord::Action::A_CREATE, "0", "this_device"));
-  records.push_back(
-      SimpleDeviceRecord(SyncRecord::Action::A_CREATE, "1", "device1"));
-
-  sync_service()->OnResolvedPreferences(records);
-
-  auto devices = brave_sync_prefs()->GetSyncDevices();
-
-  EXPECT_TRUE(DevicesContains(devices.get(), "0", "this_device"));
-  EXPECT_TRUE(DevicesContains(devices.get(), "1", "device1"));
-
-  EXPECT_CALL(*sync_client(),
-              SendSyncRecords("PREFERENCES",
-                              ContainsDeviceRecord(SyncRecord::Action::A_DELETE,
-                                                   "this_device")))
-      .Times(1);
-
-  EXPECT_CALL(*sync_client(), SendFetchSyncRecords);
-  sync_service()->OnResetSync();
-  RecordsList resolved_records;
-  auto resolved_record = SyncRecord::Clone(*records.at(0));
-  resolved_record->action = SyncRecord::Action::A_DELETE;
-  resolved_records.push_back(std::move(resolved_record));
-
-  sync_service()->OnResolvedPreferences(resolved_records);
-
+void BraveSyncServiceTest::VerifyResetDone() {
   auto devices_final = brave_sync_prefs()->GetSyncDevices();
```

Comment:
This should be moved out, test cases might not have exactly same device names

Reply:
addressed in https://github.com/brave/brave-core/compare/046dbae9503a4c53675d7abeb578cbb322ecae28..1e0c3380bf4c893c90e7624f4c09d716488c6487